package.json

```diff
@@ -62,6 +64,37 @@
     "nyc": "^15.0.0",
     "standard": "^16.0.0",
     "tap-spec": "^5.0.0",
-    "tape": "^5.0.0"
+    "tape": "^5.0.0",
+    "@semantic-release/changelog": "^6.0.1",
+    "@semantic-release/git": "^10.0.1",
+    "commitizen": "^4.2.4",
+    "cz-conventional-changelog": "^3.3.0",
+    "semantic-release": "^18.0.1"
+  },
+  "release": {
```

Comment:
I would recommend moving this to a config file, such as:
https://github.com/ExpediaGroup/steerage/blob/master/.releaserc.json

Config docs:
https://github.com/semantic-release/semantic-release/blob/master/docs/usage/configuration.md#configuration-file